pwiz_tools/Skyline/app.config

```diff
@@ -234,9 +234,15 @@
             <setting name="AreaGraphType" serializeAs="String">
                 <value>replicate</value>
             </setting>
+            <setting name="DetectionGraphType" serializeAs="String">

```

Comment:
Why would you need to change so much in this file? Have others failed to update it?